build.cake

```diff
@@ -120,72 +131,95 @@ Task("Package")
             Information("Version: {0}", RELEASE_NOTES.Version);
             Information(string.Join(Environment.NewLine, RELEASE_NOTES.Notes));
 
-			foreach (var project in GetFiles("./Source/**/*.csproj"))
-			{
-				var name = project.GetDirectory().FullPath;
-				var version = VERSION.ToString();
-				
-				if ((name.Contains("Test") && !name.Contains("TestHelpers")) || name.Contains("Example"))
-				{
-					continue;
-				}
+            foreach (var project in GetFiles("./Source/**/*.csproj"))
+            {
+                var name = project.GetDirectory().FullPath;
+                var version = VERSION;
+                
+                if ((name.Contains("Test") && !name.Contains("TestHelpers")) || name.Contains("Example"))
+                {
+                    continue;
+                }
 
                 SetReleaseNotes(project.ToString());
-							
-				DotNetCorePack(
-					name,
-					new DotNetCorePackSettings()
-					{
-						Configuration = CONFIGURATION,
-						OutputDirectory = DIR_OUTPUT_PACKAGES,
-						NoBuild = true,
-						ArgumentCustomization = aggs => aggs.Append(GetDotNetCoreArgsVersions())
-					});
-			}
+                            
+                DotNetCorePack(
+                    name,
+                    new DotNetCorePackSettings()
+                    {
+                        Configuration = CONFIGURATION,
+                        OutputDirectory = DIR_OUTPUT_PACKAGES,
+                        NoBuild = true,
+                        ArgumentCustomization = aggs => aggs.Append(GetDotNetCoreArgsVersions())
+                    });
+            }
         });
 
 // =====================================================================================================
-Task("ValidateSourceLink")
```

Comment:
Can we enhance existing cacke file without removing a lot of existing code? 
Is this file still work in appVeyor? 
Is this file usefull for local builds?

Reply:
I removed all AppVeyor stuff  in purpouse - to keep cake-file clean. We dont't have AppVeyor

Reply:
We don't have AppVeyor, but others and upstream has.  We will get merge conflicts when upstream updates this file.

So I suggest:
- to create different file for our local teamcity (and reuse all possible in both files)
- or insert code necessary to teamcity into existing pipeline